app/src/main/java/org/thoughtcrime/securesms/conversation/v2/ConversationActivityV2.kt

```diff
@@ -1234,8 +1293,9 @@ class ConversationActivityV2 : PassphraseRequiredActionBarActivity(), InputBarDe
         Toast.makeText(this, R.string.copied, Toast.LENGTH_SHORT).show()
     }
 
+    // TODO: don't need to allow new closed group check here, removed in new disappearing messages
```

Comment:
Has this been addressed?

Reply:
I have no idea what this TODO means. 

The disappearing messages seems to work in groupsv2, so not sure what else should be done